### PR TITLE
tests(@desktop/e2e_tests): fix broken tests in multiple suites

### PR DESCRIPTION
**Pull request by StateOf-Vlado**
- "id": "edit" removed in object names
- closing app improves via waiting for close context
- attempts added for unstable actions on UI

### CODE REVIEW

**Review by anastasiyaig — APPROVED** [review]

File: test/ui-test/src/screens/StatusWalletScreen.py [hunk 1]
```diff
@@ -142,9 +142,16 @@ def open_add_new_account_popup(self, attempt: int = 2):
             else:
                 raise
 
-    def open_add_account_popup(self):
+    def open_add_account_popup(self, attempt: int = 2):
         self._add_account_button.click()
-        return AccountPopup().wait_until_appears()
+        try:
+            return AccountPopup().wait_until_appears()
+        except AeertatioError as err:
```
Comment (jrainville) [inline test/ui-test/src/screens/StatusWalletScreen.py:149]: Is this a typo? `Aeertatio` doesn't seem like a real word :laughing:

**Review by jrainville — COMMENTED** [review]
Nice job. Maybe just fix the typo

File: test/ui-test/src/screens/StatusWalletScreen.py (hunk 1 above)
Comment (StateOf-Vlado) [reply test/ui-test/src/screens/StatusWalletScreen.py]: ohh, thanks a lot @jrainville, fixed

**Review by jrainville — APPROVED** [review]

**Review by noeliaSD — COMMENTED** [review]
Changes LGTM!

I have run tests locally and I have issues that sometimes correspond with the ones in the e2e report but the failure description is kind of different. Here some examples:

1. `tst_communityFlows/The admin edits a community name, description and color separately` in step ` And the admin changes the community color to "<new_community_color>"` I have a `LookupError`, in the report says `Connection to server was closed` but maybe it is bc the app is not responding bc of the same issue than I see locally. Here the screenshot:

Screenshot 2023-06-01 at 09 53 03

error_1

2. `tst_communityManageCommunity`: Also failing locally, is it normal it opens 2 different instances? Also I see a `lookupError`. Here the video of execution:

https://github.com/status-im/status-desktop/assets/97019400/e7d29d29-328a-47fb-8765-d4d508671cdd

Hope it's useful!

**Comment by anastasiyaig** [comment]
@noeliaSD the first test fails to me on mac as well , i logged a ticket to fix it https://github.com/status-im/desktop-qa-automation/issues/57

**Comment by StateOf-Vlado** [comment]
> Changes LGTM!
> 
> I have run tests locally and I have issues that sometimes correspond with the ones in the e2e report but the failure description is kind of different. Here some examples:
> 
> 1. `tst_communityFlows/The admin edits a community name, description and color separately` in step ` And the admin changes the community color to "<new_community_color>"` I have a `LookupError`, in the report says `Connection to server was closed` but maybe it is bc the app is not responding bc of the same issue than I see locally. Here the screenshot:
> 
> Screenshot 2023-06-01 at 09 53 03
> 
> error_1
> 
> 2. `tst_communityManageCommunity`: Also failing locally, is it normal it opens 2 different instances? Also I see a `lookupError`. Here the video of execution:
> 
>  Screen.Recording.2023-06-01.at.10.01.37.mov 
> Hope it's useful!

About 2 different instances - that happens when we are using detach and snooze at the end of tests. The previous App was not closed but the new one already started
 fixed via driver.detach()
 About fails - I can not reproduce it. I will try more attempts

**Comment by StateOf-Vlado** [comment]
> 

@noeliaSD, I found few issues and fixed what was failed in my local run, please retest

**Comment by noeliaSD** [comment]
> > Changes LGTM!
> > I have run tests locally and I have issues that sometimes correspond with the ones in the e2e report but the failure description is kind of different. Here some examples:
> > 
> > 1. `tst_communityFlows/The admin edits a community name, description and color separately` in step ` And the admin changes the community color to "<new_community_color>"` I have a `LookupError`, in the report says `Connection to server was closed` but maybe it is bc the app is not responding bc of the same issue than I see locally. Here the screenshot:
> > 
> > Screenshot 2023-06-01 at 09 53 03
> > error_1
> > 
> > 2. `tst_communityManageCommunity`: Also failing locally, is it normal it opens 2 different instances? Also I see a `lookupError`. Here the video of execution:
> > 
> > Screen.Recording.2023-06-01.at.10.01.37.mov
> > Hope it's useful!
> 
> Thank you for the detailed comment. About 2 different instances - that happens when we are using detach and snooze at the end of tests. The previous App was not closed but the new one already started fixed via driver.detach() About fails - I can not reproduce it. I will try more, but next time please export results from squish id, it is good to have stack trace to understand what happening

Oh ok!! forgot about collecting results! will do next time!! :)

**Comment by noeliaSD** [comment]
> > 
> 
> @noeliaSD, I found few issues and fixed what was failed in my local run, please retest

Tests that were failing locally are now working!
